### PR TITLE
Fixes typeinfo error

### PR DESCRIPTION
**Pull request by jeromegn**
Also should build on travis, can't really test with my repo because of hard-coded "augustoroman", and the very nature of golang package imports :)

We should see if this works when travis builds this PR.

### CODE REVIEW

**Comment by jeromegn** [comment]
@augustoroman I don't think I can make that example test pass. The trailing new line is not handled well afaict. There's no way to add a new line to the desired output because of gofmt (even if we didn't run it, somebody is bound to save and run it automatically.)

Could simplify the example maybe. Or not test it?

**Comment by augustoroman** [comment]
It's not the trailing newline, it's the space between function and the parens:
```
// hello.js:3> function() { return "x" }
```
should be
```
// hello.js:3> function () { return "x" }
                       ^
```

**Comment by augustoroman** [comment]
This is an awesome improvement!  I'm a little concerned about the -fno-rtti flag required, since that's going into the actual code and isn't required currently (for me).  I'm gonna look into this tonight and see if I can't figure out a way to make it consistent.

**Comment by jeromegn** [comment]
Whoops, I missed that space entirely.

Concerning the flag: your v8 might have been compiled with run-time type info? Not sure if that changes anything for cgo.

File: v8.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ package v8
 // #include <stdlib.h>
 // #include <string.h>
 // #include "v8_c_bridge.h"
-// #cgo CXXFLAGS: -I${SRCDIR} -I${SRCDIR}/include -std=c++11
```
Comment (augustoroman) [inline v8.go]: don't remove the two `-I` flags -- they are still needed, it's just that you are getting them from v8_create.go.

Add `-fno-rtti` to v8_create.go as well.

Finally, update the README to include instructions for extracting the ruby gem on osx.

**Comment by jeromegn** [comment]
Done! I did my best with the README, it works for more than just macos btw.

**Pull request merged**